docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -235,42 +235,77 @@ containing total concentration of volcanic ash:
     >>> print regional_ash.summary(shorten=True)
     VOLCANIC_ASH_AIR_CONCENTRATION / (g/m3) (latitude: 214; longitude: 584)
 
-One of the key limitations to the AreaWeighted regridding scheme is that the two
-input grids must be defined in the same coordinate system and both must contain
-monotonic, bounded, 1D spatial coordinates.
+One of the key limitations of the :class:`~iris.analysis.AreaWeighted`
+regridding scheme is that the two input grids must be defined in the same
+coordinate system as each other. Both must also contain monotonic, bounded,
+1D spatial coordinates.
 
 .. note::
 
-    The area weighted scheme requires spatial areas, therefore the longitude and
-    latitude coordinates must be bounded. In this case, we can simply guess bounds
-    based on the point values, but this step will is not necessary if the cube being
-    worked with is already bounded:
+    The :class:`~iris.analysis.AreaWeighted` regridding scheme requires spatial
+    areas, therefore the longitude and latitude coordinates must be bounded.
+    If the longitude and latitude bounds are not specified in the cube we can
```

Comment:
`If the longitude and latitude bounds are not defined in the cube then we can ...`